paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountEmitters.kt

```diff
@@ -52,6 +52,7 @@ internal fun USBankAccountEmitters(
                 context = context,
                 merchantName = merchantName,
                 isSaveForFutureUseSelected = saved,
+                isSetupFlow = !usBankAccountFormArgs.isPaymentFlow,
```

Comment:
We've got a lot of glue code like this that isn't tested. It's a little scary! Is there a way to add a test for it?